tests/DependencyInjection/StofDoctrineExtensionsExtensionTest.php

```diff
@@ -10,6 +10,9 @@
 
 class StofDoctrineExtensionsExtensionTest extends TestCase
 {
+    /**
+     * @return list<array{string}>
```

Comment:
I suggest documenting it as `iterable<...>` as data providers don't care about getting a list but an iterable.